app/modules/entities/views/work_li.js

```diff
@@ -1,23 +1,36 @@
 import { invertAttr, isOpenedOutside } from 'lib/utils'
 import workLiTemplate from './templates/work_li.hbs'
+import workLiTemplateTable from './templates/work_li_table.hbs'
 import '../scss/work_li.scss'
+import '../scss/work_li_table.scss'
+import { localStorageProxy } from 'lib/local_storage'
 
 export default Marionette.ItemView.extend({
-  template: workLiTemplate,
   className () {
     const prefix = this.model.get('prefix')
     if (this.wrap == null) this.wrap = this.options.wrap
     const wrap = this.wrap ? 'wrapped wrappable' : ''
-    return `workLi entity-prefix-${prefix} ${wrap}`
+    const workLiDisplay = this.display === 'entitiesTable' ? 'workLiTable' : 'workLi'
+    return `${workLiDisplay} entity-prefix-${prefix} ${wrap}`
   },
 
   attributes () {
     // Used by deduplicate_layout
     return { 'data-uri': this.model.get('uri') }
   },
 
+  getTemplate () {
+    if (this.display === 'entitiesTable') {
+      return workLiTemplateTable
+    } else {
+      return workLiTemplate
+    }
+  },
+
   initialize () {
     let preventRerender
+
+    this.display = localStorageProxy.getItem('entitiesDisplay') || 'entitiesCascade'
```

Comment:
this `entitiesDisplay` local storage key is inconsistent with the `entities:display`, so one or the other is wrong